packages/@aws-cdk/aws-apigateway/test/base-path-mapping.test.ts

```diff
@@ -77,6 +101,78 @@ describe('BasePathMapping', () => {
     }).toThrowError(/base path may only contain/);
   });
 
+  test('throw error for basePath starting with /', () => {
```

Comment:
APIGateway definitely forbids this?

Reply:
Verified this behaviour via AWS Console. Screenshot attached below - 
![Screenshot 2022-12-16 at 1 22 20 AM](https://user-images.githubusercontent.com/7585096/207954286-19e8c46e-f3a4-4e2e-b3ed-ffaad2598dc9.png)